frontend/src/hooks/useInvalidateSecretQueries.ts

```diff
@@ -0,0 +1,89 @@
+import { useCallback } from "react";
+import { useQueryClient } from "@tanstack/react-query";
+
+import { dashboardKeys } from "./api/dashboard/queries";
+import { commitKeys } from "./api/folderCommits/queries";
+import { secretApprovalRequestKeys } from "./api/secretApprovalRequest/queries";
+import { secretKeys } from "./api/secrets/queries";
+import { secretSnapshotKeys } from "./api/secretSnapshots/queries";
+
+export function useInvalidateSecretQueries(projectId: string) {
```

Comment:
Functionality was part of `SecretListView`. Now used in `useHandleSecretOperation` to provide uniform invalidation after changes.